### PR TITLE
DOC-6682: Added durability to buckets

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/learn/pages/data/durability.adoc [hunk 1]
```diff
@@ -110,6 +110,7 @@ Clients that attempt to write to a key whose value is undergoing a durable write
 
 [#durable-write-lifecycle]
 The lifecycle of a durable write is shown by the following, annotated diagram; which shows multiple clients' attempts to update and read a value, with the timeline moving from left to right.
+The scenario assumes that the required durability-level is client-specified, with no level having been assigned to the bucket: however, the lifecycle is the same regardless of whether the prevailing durability level has been imposed by client or bucket.
```
Comment (owend74) [inline modules/learn/pages/data/durability.adoc]: I'm not sure this sentence is required.  And arguably makes it a bit more confusing.

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 2]
```diff
@@ -48,262 +54,859 @@ curl -X POST -u <administrator>:<password>
...
+
+** A single parameter that is valid for Couchbase Server Community Edition only; this being xref:rest-api:rest-bucket-create.adoc#indexcompactionmode[indexCompactionMode].
```
Comment (owend74) [inline modules/rest-api/pages/rest-bucket-create.adoc:110]: Just to confirm this is Community Edition - and not Enterprise?

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 3]
```diff
@@ -48,262 +54,859 @@ curl -X POST -u <administrator>:<password>
...
 If this parameter is incorrectly specified, an error-notification such as the following is returned: `"databaseFragmentationThreshold[percentage]":"database fragmentation must be an integer"`.
-+
+
 This parameter is ignored if `autoCompactionDefined` is `false` (which is its default value).
 
-* `databaseFragmentationThreshold[size]`.
+[#example-databasefragmentationthresholdpercentage-create]
+==== Example: Specifying a Data Fragmentation Threshold as a Percentage, when Creating
+
+The following example establishes a value for `databaseFragmentationThreshol[percentage]`, and for all other auto-compaction-related parameters, in its creation of a new bucket, named `testBucket`:
```
Comment (owend74) [inline modules/rest-api/pages/rest-bucket-create.adoc]: databaseFragmentationThreshol is it missed a 'd'?

File: modules/learn/pages/data/durability.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/learn/pages/data/durability.adoc]: Ok, thanks. I've removed.

File: modules/rest-api/pages/rest-bucket-create.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-bucket-create.adoc:110]: Yes, index compaction is community edition only.

File: modules/rest-api/pages/rest-bucket-create.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-bucket-create.adoc]: Thanks, have fixed.

**Comment by tonyjhillman** [comment]
Have included these changes now. However, perhaps I will also include DOC-6883 in this PR.

**Comment by tonyjhillman** [comment]
Approved by Daniel Owen.

**Pull request merged**